linux_os/guide/system/network/network-iptables/iptables_activation/set_loopback_traffic/rule.yml

```diff
@@ -23,21 +23,24 @@ identifiers:
 references:
     cis@sle12: 3.5.2.1
     cis@sle15: 3.5.2.6,3.5.3.2.2
+    cis@ubuntu2204: 3.5.3.2.2
     pcidss: Req-1.4.1
 
 warnings:
     - general: |-
-        Changing firewall settings while connected over network can 
+        Changing firewall settings while connected over network can
         result in being locked out of the system.
 
 ocil_clause: 'loopback traffic is not configured'
 
 ocil: |-
-    Verify that the loopback interface is configured:
+    Run the following commands and verify output:
+    <pre>
+    # iptables -L INPUT -v -n | grep lo | grep ACCEPT
+    </pre>
     <pre>
-    # nft list ruleset | awk '/hook input/,/}/' | grep 'ip saddr'
```

Comment:
Switching to iptables here concerns me. I believe we should add Jinja condition for these products that want iptables.

Reply:
shouldn't nftables rules be under: `linux_os/guide/system/network/network-nftables/` ?

The bash itself is about iptables and not nft

Reply:
I agree; we will keep `iptables` here.